src/components/containers/notification/Notification.js

```diff
@@ -146,6 +146,13 @@ function renderCallsToAction(callsToAction) {
       {callsToAction.map((callToAction, index) => {
         const buttonStyleType = index === 0 ? 'primary' : 'default';
 
+        if (React.isValidElement(callToAction)) {
```

Comment:
I think we'll let this go for now, but I think a better solution would be to pull this out into a special Notification component in the ``Patterns`` section. thoughts @Darrken ?